src/Toolkit/Toolkit/GeoViewController.cs

```diff
@@ -0,0 +1,279 @@
+            ConnectedView?.IdentifyLayersAsync(screenPoint, tolerance, returnPopupsOnly, cancellationToken) ??
+                Task.FromResult<IReadOnlyList<IdentifyLayerResult>>(new List<IdentifyLayerResult>().AsReadOnly());
+        
+        /// <inheritdoc cref="GeoView.IdentifyLayerAsync(Layer, Point, double, bool, CancellationToken)"/>
+        public virtual Task<IdentifyLayerResult> IdentifyLayerAsync(Layer layer, Point screenPoint, double tolerance, bool returnPopupsOnly = false, CancellationToken cancellationToken = default) =>
+            ConnectedView?.IdentifyLayerAsync(layer, screenPoint, tolerance, returnPopupsOnly, cancellationToken) ??
+                Task.FromResult<IdentifyLayerResult>(null!);
+
+        /// <inheritdoc cref="GeoView.IdentifyGraphicsOverlaysAsync(Point, double, bool, long)" />
+        public virtual Task<IReadOnlyList<IdentifyGraphicsOverlayResult>> IdentifyGraphicsOverlaysAsync(Point screenPoint, double tolerance, bool returnPopupsOnly = false, long maximumResultsPerOverlay = 1) =>
+            ConnectedView?.IdentifyGraphicsOverlaysAsync(screenPoint, tolerance, returnPopupsOnly, maximumResultsPerOverlay) ??
+                Task.FromResult<IReadOnlyList<IdentifyGraphicsOverlayResult>>(new List<IdentifyGraphicsOverlayResult>().AsReadOnly());
+
+        /// <inheritdoc cref="GeoView.IdentifyGraphicsOverlaysAsync(Point, double, bool, long)" />
+        public virtual Task<IdentifyGraphicsOverlayResult> IdentifyGraphicsOverlayAsync(GraphicsOverlay overlay, Point screenPoint, double tolerance, bool returnPopupsOnly = false, long maximumResults = 1) =>
+            ConnectedView?.IdentifyGraphicsOverlayAsync(overlay, screenPoint, tolerance, returnPopupsOnly, maximumResults) ??
+                Task.FromResult<IdentifyGraphicsOverlayResult>(null!);
+        #endregion Identify
+
+        #region Callouts
+
+        /// <inheritdoc cref="GeoView.DismissCallout()" />
+        public virtual void DismissCallout() => ConnectedView?.DismissCallout();
+
+        /// <inheritdoc cref="GeoView.ShowCalloutAt(Geometry.MapPoint, CalloutDefinition)" />
+        public virtual void ShowCalloutAt(Esri.ArcGISRuntime.Geometry.MapPoint location, CalloutDefinition definition) => ConnectedView?.ShowCalloutAt(location, definition);
+
+        /// <inheritdoc cref="GeoView.ShowCalloutForGeoElement(GeoElement, Point, CalloutDefinition)" />
+        public virtual void ShowCalloutForGeoElement(GeoElement element, Point tapPosition, CalloutDefinition definition) => ConnectedView?.ShowCalloutForGeoElement(element, tapPosition, definition);
+        #endregion Callouts
+
+#if MAUI
+        /// <summary>
+        /// Identifies the <see cref="GeoViewController"/> attached property.
+        /// </summary>
+        public static BindableProperty GeoViewControllerProperty =
```

Comment:
This BindableProperty field can be readonly.
```suggestion
        public static readonly BindableProperty GeoViewControllerProperty =
```